cts/scheduler/summary/bundle-order-stop-on-remote.summary

```diff
@@ -140,8 +140,8 @@ Executing Cluster Transition:
   * Resource action: galera-bundle-docker-2 monitor=60000 on database-2
   * Resource action: galera-bundle-2 start on controller-1
   * Resource action: redis           notify on redis-bundle-0
-  * Resource action: redis           notify on redis-bundle-1
   * Resource action: redis           notify on redis-bundle-2
+  * Resource action: redis           notify on redis-bundle-1
```

Comment:
Previous to this change, order_instance_starts_stops() sorted the instances by number since each was ordered only relative to the next, while now they're ordered relative to every later instance. It just so happens that the instances are in a different order when notifications get created.

Probably not worth doing anything about. There are some cases where we order things just if a simulation is in progress, but that's mainly for hash tables where the order would otherwise be different every time.